wiki/Client/File_formats/osr_(file_format)/de.md

```diff
@@ -0,0 +1,101 @@
+# .osr (Dateiformat)
+
+**.osr** ist ein Dateiformat mit Informationen über ein osu!-Replay. Für die Verwendung muss sich die zum Replay gehörende Beatmap im Ordner "Songs" befinden.
+
+## Datentypen
+
+| Name | Länge in Bytes | Beschreibung |
+| :-- | :-- | :-- |
+| Byte | 1 | Ein einzelner 8-Bit-Wert. |
```

Comment:
I think leaving the data type names in their original form is fine, since they are pretty common in German too (maybe except when used in a text).

Reply:
I agree. I would even say that they *should not* be translated, as they're common terms in informatics (excluding maybe LEB128, but that one is an abbreviation of other common terms).